src/grg_sphinx_theme/theme/grg_sphinx_theme/static/styles/components/footer.css

```diff
@@ -35,13 +35,13 @@ ul.footer-section li a:visited {
   text-decoration: none;
 }
 
-ul.footer-section li i {
+ul.footer-section li a svg {
```

Comment:
Can you have both to be backward compatible ? Just in case we need to go back to an older version and for the older build